core/lib/src/response/responder.rs

```diff
@@ -321,6 +327,18 @@ impl<'r, R: Responder<'r> + Send + 'r, E: Responder<'r> + fmt::Debug + Send + 'r
     }
 }
 
+/// Responds with the wrapped `Responder`.
+impl<'r, L: Responder<'r> + Send + 'r, R: Responder<'r> + Send + 'r> Responder<'r> for either::Either<L, R> {
+    fn respond_to(self, req: &'r Request<'_>) -> response::ResultFuture<'r> {
+        Box::pin(async move {
```

Comment:
We shouldn't need to make another box here, since each branch already creates a boxed future

Reply:
Isn't that also the case in other implementations in the same module?

Reply:
Quick check says yes. I'll do that as a separate PR.